### PR TITLE
T360753: Gallery row update

### PR DESCRIPTION
**Pull request by medied**
https://phabricator.wikimedia.org/T360753

Demo: https://wikimedia.github.io/wikipedia-preview/T360753-gallery-images/articles/english.html

* Updates preview layout 
* Loading state will be maintained until both preview and media data are available
* Storybook and tests updated accordingly

---

image

### CODE REVIEW

**Comment by hueitan** [comment]
the gallery height looks a little bit bigger than usual?
image

**Comment by medied** [comment]
> the gallery height looks a little bit bigger than usual?

I know right? And yet it's still about 10px smaller than the design in Figma. I plan to sync with Sudhanshu to go over the pixel details

File: style/preview.less [hunk 1]
```diff
@@ -20,18 +20,18 @@
 
 	&-header {
 		display: flex;
-		height: 78px;
+		height: 52px;
 		line-height: 62px;
-		margin-top: 12px;
-		margin-bottom: 12px;
```
Comment (hueitan) [inline style/preview.less:26]: | the header with thumbnail top margin/padding doesn't look good on | header without thumbnail looks fine | header get cutoff in smaller screen (i think it doesn't happen before) |
| ---| --- | --- |
| image| image |image |

**Review by hueitan — COMMENTED** [review]
| Before | After |
| --- | --- |
| image | image |

\
seems like loading bar are getting smaller, why is that?

**Comment by hueitan** [comment]
One more thing, should the max-height be the same for case with and without thumbnail, and more content scrollable

currently it doesn't look good to me
| with thumbnail | without thumbnail |
| --- | --- |
| image | image |

**Comment by medied** [comment]
Thanks @hueitan, I synced with Sudhanshu today and went over all your feedback, please check the latest style update

**Review by hueitan — APPROVED** [review]
merge it now

**Pull request merged**